source/connect/favorite-connections/import-export-ui/import.txt

```diff
@@ -0,0 +1,57 @@
+.. _compass-import-connections-ui:
+
+==============================================
+Import Favorite Connections in |compass-short|
+==============================================
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+You can import favorite connections in |compass|. This lets you use saved 
+connections from other workspaces or connections used by other team members.
+
+Prerequisite
+------------
+
+Before you import favorite connections, you must :ref:`export the
+connections <compass-export-connections-ui>`.
+
+Procedure
+~~~~~~~~~
+
+To import favorite connections from a JSON file into |compass-short|:
+
+.. procedure::
+    :style: normal
+    
+    .. step:: On the Connect screen, hover over :guilabel:`Saved Connections` 
+        in the left-side navigation and click the :guilabel:`Ellipses (...)` 
+        button.
+
+    .. step:: Select :guilabel:`Import saved connections`.
+
+    .. step:: Under :guilabel:`Source File`, click :guilabel:`Select a file`. 
+        
+    .. step:: Select the exported JSON file you want to import.
+
+    .. step:: If your file is encrypted with a password, enter the passphrase 
+        under :guilabel:`Decryption Password`.
+    
+    .. step:: Select the connection names you want to 
+        import. 
+        
+        If the connection name already exists under your saved connections, 
+        |compass-short| overwrites the existing connection with the imported 
+        connection.
+
+    .. step:: Click :guilabel:`Import`.
+        
+Results
+~~~~~~~
+
+After the connections are imported, you will see them on the left-side 
```

Comment:
[nit, minimalism]

Maybe: "After the connections are imported, they appear on the left-side..."

Reply:
Updated.